src/v2/config.rs

```diff
@@ -16,8 +17,7 @@ impl Config {
     /// Initialize `Config` with default values.
     pub fn default(handle: &reactor::Handle) -> Self {
         Self {
-            config: hyper::client::Client::configure()
-                .connector(hyper_rustls::HttpsConnector::new(4, handle)),
+            config: hyper::client::Client::builder().build(hyper_rustls::HttpsConnector::new(4)),
```

Comment:
That's the magic 4 here?

Reply:
Number of DNS threads internally spawned by the TLS connector. I'll make this an explicit variable with a proper name.